media/pages/sk/podmienky-pouzivania/page.html

```diff
@@ -44,13 +44,14 @@ <h3 id="definicie-2">
 
 <p id="definicie-2-1-a">
 <span>a)</span>
-Podmienky sú tieto podmienky používania portálu www.chcemvediet.sk; 
+Podmienky sú tieto podmienky používania portálu
+<a href="https://chcemvediet.sk/">www.chcemvediet.sk</a>;
 </p>
 
 <p id="definicie-2-1-b">
 <span>b)</span>
-Portálom sa rozumie internetový portál www.chcemvediet.sk vrátane všetkých 
-jeho podstránok, súborov a funkcii; 
+Portálom sa rozumie internetový portál <a href="https://chcemvediet.sk/">www.chcemvediet.sk</a>
+vrátane všetkých jeho podstránok, súborov a funkcií;
```

Comment:
@martinmacko47 Hrozne sa mi nepaci slovicko "uploadnuť" v sekcii 5.11. Radsej by som ho prelozil do slovenčiny "nahrať" alebo "načítať"

Reply:
Kludne by som slovo "uploadnuť" nechal. IMHO je to zrozumitelnejsie, nez "nahrať", kedze "nahrať" moze znamenat kadeco a slovo "uploadnuť" sa pouziva iba v jednom vyzname (aj ked nie je spisovne).

Reply:
:ok: